coba/tests/test_evaluators_online.py

```diff
@@ -658,8 +658,17 @@ def request(self, context, actions, request):
     def test_batched_request_continuous(self):
         class TestLearner:
             def request(self,context,actions,request):
-                if isinstance(context,BatchType): raise Exception()
-                return .5
+                if isinstance(context,BatchType):
+                    raise Exception()
+                return 0.5
+
+            def predict(self, context, actions):
```

Comment:
Struggling to make this test pass. 
The processing thinks it's of `AX` format and then always fills in `None`s for the probability. I haven't worked with continuous actions before and I am not quite sure about all the different formats and in the `SafeLearner`
Any advice, @mrucker?

<img width="1157" alt="Screenshot 2023-11-06 at 12 32 28 PM" src="https://github.com/VowpalWabbit/coba/assets/1749498/674a44ac-6673-455f-a3b2-4117ad85425f">